plugin/pulp/plugin/serializers.py

```diff
@@ -0,0 +1,2 @@
+# All Serializers in platform are potentially useful to plugin writers
```

Comment:
I disagree. The plugin API is everything that starts with pulp.plugin. We also haven't yet declared the plugin API, so even if I did add the comment it's just as likely to be wrong as this is.